narwhal/node/src/main.rs

```diff
@@ -71,10 +71,6 @@ async fn main() -> Result<(), eyre::Report> {
                 .args_from_usage("--workers=<FILE> 'The file containing worker information'")
                 .args_from_usage("--parameters=[FILE] 'The file containing the node parameters'")
                 .args_from_usage("--store=<PATH> 'The path where to create the data store'")
-                .subcommand(SubCommand::with_name("primary")
-                    .about("Run a single primary")
```

Comment:
I believe we need this so we can run a `primary` node - the arg is indeed not needed

Reply:
Reverted.